src/reporting/claim-reporting-fees-forked-market.js

```diff
@@ -0,0 +1,237 @@
+      initialReporterRedeem: new BigNumber(0),
+      all: new BigNumber(0),
+    },
+  };
+
+  var redeemableContracts = [];
+  var i;
+  if (p.forkedMarket) {
+    if (p.forkedMarket.crowdsourcers) {
+      for (i = 0; i < p.forkedMarket.crowdsourcers.length; i++) {
+        redeemableContracts.push({
+          address: p.forkedMarket.crowdsourcers[i].crowdsourcerId,
+          needsFork: p.forkedMarket.crowdsourcers[i].needsFork,
+          type: contractTypes.DISPUTE_CROWDSOURCER,
+        });
+      }
+    }
+    if (p.forkedMarket.initialReporter && p.forkedMarket.initialReporter.initialReporterId) {
+      redeemableContracts.push({
+        address: p.forkedMarket.initialReporter.initialReporterId,
+        needsFork: p.forkedMarket.initialReporter.needsFork,
+        type: contractTypes.INITIAL_REPORTER,
+      });
+    }
+  }
+
+  async.eachLimit(redeemableContracts, PARALLEL_LIMIT, function (contract, nextContract) {
+    switch (contract.type) {
+      case contractTypes.DISPUTE_CROWDSOURCER:
+        if (contract.needsFork) {
+          api().DisputeCrowdsourcer.forkAndRedeem(assign({}, payload, {
+            tx: {
+              to: contract.address,
+              estimateGas: p.estimateGas,
+            },
+            onSent: function () {},
+            onSuccess: function (result) {
+              if (p.estimateGas) {
+                result = new BigNumber(result, 16);
+                gasEstimates.crowdsourcerForkAndRedeem.push({address: contract.address, estimate: result});
+                gasEstimates.totals.crowdsourcerForkAndRedeem = gasEstimates.totals.crowdsourcerForkAndRedeem.plus(result);
+              } else {
+                successfulTransactions.crowdsourcerForkAndRedeem.push(contract.address);
+              }
+              // console.log("Forked and redeemed crowdsourcer", contract.address);
+              nextContract();
+            },
+            onFailed: function () {
+              failedTransactions.crowdsourcerForkAndRedeem.push(contract.address);
+              // console.log("Failed to forkAndRedeem crowdsourcer", contract.address);
+              nextContract();
+            },
+          }));
+        } else {
+          api().DisputeCrowdsourcer.redeem(assign({}, payload, {
+            _redeemer: p.redeemer,
+            tx: {
+              to: contract.address,
+              estimateGas: p.estimateGas,
+            },
+            onSent: function () {},
+            onSuccess: function (result) {
+              if (p.estimateGas) {
+                result = new BigNumber(result, 16);
+                gasEstimates.crowdsourcerRedeem.push({address: contract.address, estimate: result});
+                gasEstimates.totals.crowdsourcerRedeem = gasEstimates.totals.crowdsourcerRedeem.plus(result);
+              } else {
+                successfulTransactions.crowdsourcerRedeem.push(contract.address);
+              }
+              // console.log("Redeemed crowdsourcer", contract.address);
```

Comment:
Remove debugging comments?

Reply:
Yeah, I considered doing that, but I think I'll leave them in for now in case we want to use them later for debugging.